python/rapidsmpf/rapidsmpf/shuffler.pyx

```diff
@@ -122,7 +124,7 @@ cdef class Shuffler:
         """
         return self._comm
 
-    def insert_chunks(self, chunks):
+    def insert_chunks(self, chunks, bool grouped = False):
```

Comment:
Do we have a good reason for having a single Python method (`insert_chunks`) with a boolean flag, rather than having two methods to match the C++ API (`insert`, `insert_grouped`)? If there aren't strong reasons I'd probably vote for consistency with the C++ API (although I'm noticing now we're already inconsistent: `insert_chunks` vs `insert`.)

Reply:
@TomAugspurger IMO, the two methods have different implementations in C++ side. It was more cleaner that way rather than branching out within a single method. But in python side, if we had two methods, both would have identical impl, because `insert` and `insert_grouped` have identical signatures. 
That's why I chose a flag in the python side.

Reply:
Works for me. I just wanted to confirm we weren't violating a rule of the Python API being a strict wrapper around the C++ API, like pylibcudf does.